test/cypress/e2e/block.cy.ts

```diff
@@ -1,12 +1,93 @@
+import { blockTestHashes } from './hashes/blockTestHashes';
+
+const {
+  blockHash,
+  parentHash,
+  validator,
+  blockHeight,
+  deployHash,
+  truncatedParentHash,
+  truncatedDeployHash,
+  truncatedValidatorHash,
+  truncatedTransfer,
+  transferDeployHash,
+  truncatedBlockHash,
+} = blockTestHashes();
+
 describe('Block Page', () => {
-  it('can visit at /block/:blockHeight', () => {
-    const blockHeight = 1;
+  beforeEach(() => {
+    cy.visit(`/block/${blockHash}`);
+  });
 
-    cy.visit(`/block/${blockHeight}?type=height`);
+  const rawDataHeading = 'AccordionItemHeading';
+  const rawDataButton = 'AccordionItemButton';
 
+  it('can visit at /block/:blockHeight', () => {
```

Comment:
Can you add tests for both mobile screen size and desktop.  It's more important for this test as there are two largely different UI's base on the screensize